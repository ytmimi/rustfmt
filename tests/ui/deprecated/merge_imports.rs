// rustfmt-merge_imports: true
