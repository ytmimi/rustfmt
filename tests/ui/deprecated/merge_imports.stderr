Warning: the `merge_imports` option is deprecated. Use `imports_granularity="Crate"` instead
